export.py

```diff
@@ -41,7 +41,7 @@ def json_to_html(json):
 {body}
 </article>
 """.format(
-        subject=json['subject'] or json['date'],
+        subject=json['subject'] or json['eventtime'],
```

Comment:
Why the switch from date to eventtime everywhere? Has LJ changed their output format?